### ISSUE TITLE
Describe simulation more precisely in Fig 4

### ISSUE THREAD

**Issue by jeromekelleher** [tskit-dev/what-is-an-arg-paper#319]
Currently say this:

> Fig. 4 shows a series of successive steps of simplification, starting with a sample-resolved but unsimplified gARG, simulated under a backward-time Wright-Fisher model.

We should say, what do we actually store in the WF model simulation? I don't think stacking up terminology is helping here, we just state how we performed the simulation.

This makes the problems in #267 go away.

**Comment by jeromekelleher** [comment]
Also comments from Andrew:
> (a) How the graph in part A is generated is somewhat glossed over. It's mentioned that it's a backward-time Wright-Fisher model, but why does that lead to the structure seen in A? For example, we see node i has only one parent and one child, while node e has two parents and two children. This is not classically possible under the event-based formulation of the ARG (which uses continuous time), which is the only data-generating process for generating an ARG that is discussed in this manuscript.

**Comment by hyanwong** [comment]
That's a good point. Here we are using the 2-RE-node formulation, which could reasonably be seen as confusing. Should we switch to the 1-RE-node version, or direct the reader to the appendix where we discuss the difference.

In the 1-RE-node version, we would need to represent the diamond somehow, e.g. by placing 2 separate edges between parent and child. Coupled with the definition of an edge as _p, c, I_, this would imply that we can allow multiple edges between the same parent and child, each with their own set of intervals. I guess that's fine though: we don't say that each edge requires a unique _p, c_ pair anywhere.

**Comment by jeromekelleher** [comment]
We don't need to change the formulation, we just need to *describe* what we're doing.

**Comment by hyanwong** [comment]
Yes, but this is the first time the reader will have encountered the 2-RE-node representation, which does look a bit weird to the uninitiated. I can see why it's not at all clear to the reader which nodes are the recombination nodes, here.

**Comment by jeromekelleher** [comment]
So we need to explain what's going on. 

Changing to 1RE node is a minor detail. Not explaining what's going on is a big issue - let's fix the big issues *first* and then address the small details later, if they still need to be addressed.

**Comment by hyanwong** [comment]
> So we need to explain what's going on.

Yes. But I think that means explaining the 2-RE-node concept here, that's all (rather than introducing it in the appendix). Do we want to do that?

**Comment by hyanwong** [comment]
In particular, re Andrew's comments, 

> node i has only one parent and one child, while node e has two parents and two children

This wouldn't happen in a 1-RE-node encoding, right?

**Comment by jeromekelleher** [comment]
Leave it with me. We need a paragraph describing the simulation. We also want to touch on his point to clarify, to show that both approaches are possible (WH ARG uses one RE node)

**Comment by hyanwong** [comment]
Thanks. I assume it would be interesting to see what Fig 4 would look like with the 1-RE-node encoding, anyway (we have code to do the conversion easily enough, I think?

**Comment by jeromekelleher** [comment]
Let's not, please. We have enough to do and this would require rewriting a bunch of text.

**Comment by hyanwong** [comment]
> Let's not, please. We have enough to do and this would require rewriting a bunch of text.

Yeah, diamonds might be tricky too.
